.github/workflows/super-linter.yml

```diff
@@ -27,5 +27,8 @@ jobs:
       - name: Super-linter
         uses: super-linter/super-linter@12150456a73e248bdc94d0794898f94e23127c88 # v7.4.0
         env:
+          DOTNET_GENERATE_ASPNET_CERTIFICATE: false
```

Comment:
please put this into https://github.com/grafana/docker-otel-lgtm/blob/main/.github/super-linter.env where it will be used for local runs as well